src/lib/helpers/package.ts

```diff
@@ -249,7 +249,9 @@ export const handleInstallation = async ({
   version,
 }: HandleInstallationArg): Promise<void> => {
   const installAndMoveToDependencyPath = async (): Promise<void> => {
+    await rm(dependencyTmpPath, { recursive: true, force: true });
     await installDependency();
+    await rm(dependencyPath, { recursive: true, force: true });
```

Comment:
is it a dir? or a file? check if `--rf` is needed at all

Reply:
this is a dir. you can see a recreate it on the next line. Dirs do not remove without `recursive: true`. `force: true` on the other hand makes it so it doesn't fail if there is no directory to remove

Reply:
i'd name it as `dependencyDir` then, so it is clear